docs/src/examples/components/Chat/Types/ChatExample.shorthand.tsx

```diff
@@ -6,56 +6,41 @@ const janeAvatar = {
   status: { color: 'green', icon: 'check' },
 }
 
+const content = (
+  <div>
+    Sure! Let's try one of these places:<br />
+    <a href="#">www.goodFood1.com</a>,<br />
+    <a href="#">www.goodFood2.com</a> or<br />
+    <a href="#">www.goodFood3.com</a>
+  </div>
+)
+
 const items = [
-  {
-    content: (
-      <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Hi"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Would you like to grab a lunch?"
-        author="John Doe"
-        timestamp="Yesterday, 10:16 PM"
-        mine
-      />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Sure! Let's try the new place downtown"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    ),
-  },
-  {
-    content: <Divider content="Today" type="primary" important />,
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Let's have a call"
-        author="John Doe"
-        timestamp="Today, 11:15 PM"
-        mine
-      />
-    ),
-  },
-]
+  <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />,
+  <Chat.Message
+    content="Hi"
+    author="Jane Doe"
+    timestamp="Yesterday, 10:15 PM"
+    avatar={janeAvatar}
+  />,
+  <Chat.Message
+    content="Would you like to grab a lunch?"
+    author="John Doe"
+    timestamp="Yesterday, 10:16 PM"
+    mine
+  />,
+  <Chat.Message
+    content={{ content }}
+    author="Jane Doe"
+    timestamp="Yesterday, 10:15 PM"
+    avatar={janeAvatar}
+  />,
+  <Divider content="Today" type="primary" important />,
+  <Chat.Message content="Let's have a call" author="John Doe" timestamp="Today, 11:15 PM" mine />,
+].map((chatItemContent, index) => ({
+  key: `chat-item-${index}`,
```

Comment:
We should not use antipatterns in examples, [Index as a key is an anti-pattern](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
proposal?
- we cannot import other libraries in examples in order to generate keys
- we could introduce the keys manually for each item but that'd be sad
- we could have a global counter (to this file) to hold the seed for our key generation and just increment it.

Reply:
in reality, messages retrieved from a backend will have some kind if ID. can you just manually add an ID to the  example data?

Reply:
> we could introduce the keys manually for each item but that'd be sad

It's a normal way, my opinion that we should keep examples maximally simple.

```jsx
{
    content: (
      <Chat.Message
        content="Let's have a call"
        author="John Doe"
        timestamp="Today, 11:15 PM"
        mine
      />
    ),
    key: 'message-today-11-15',
  },
```

May be it will be not so short, but it will be more obvious.

Reply:
done, thanks guys